spec/facility_spec.rb

```diff
@@ -23,4 +23,180 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe "#register_vehicle" do
+    before(:each) do
+      @facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+      @facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
+      @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+      @facility_1.add_service('Vehicle Registration')
+    end
+
+    it "can register a vehicle at the appropriate facility" do
+      expect(@cruz.registration_date).to eq (nil)
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
```

Comment:
I would probably take this out of an expect statement and make it a command query like
```ruby
@facility_1.register_vehicle(@cruz)
```
Then your expect on line 42 is testing that your `register_vehicle(vehicle)` method is doing what you want.